bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
> We'd have to make eslint a (peer) dependency, which (in my opinion) would be just fine, as the whole purpose of this module is to work with it.

Been meaning to do this anyway. :+1:

> We'd have to require a part of eslint's internals that are not part of their public api, when used as a lib (as in require('eslint')).

Maybe someone from the ESLint team could give us some insight on a better way to do this or at least give us an idea of how likely it is that this would lead to issues in the future. Cc @nzakas, @gyandeeps, @mysticatea, @ilyavolodin (top 4 code contributors on the project)